spinalcordtoolbox/cropping.py

```diff
@@ -38,9 +38,9 @@ def _get_max_value(input, dim):
             # If empty, return dim+1 (corresponds to the maximum of the given dimension, e.g. nx)
             if input is None:
                 return dim + 1
```

Comment:
to be consistent, this should be dim (not dim+1), right?

Reply:
Yes, you're right! Thx